frameworks/cassandra/tests/test_tls.py

```diff
@@ -126,13 +126,13 @@ def test_tls_connection(cassandra_service, dcos_ca_bundle):
 @pytest.mark.tls
 @pytest.mark.sanity
 def test_tls_recovery(cassandra_service, service_account):
-    pod_list = sdk_cmd.svc_cli(
+    rc, stdout, _ = sdk_cmd.svc_cli(
```

Comment:
nit. If we don't user `rc` we should just have `_`.

Reply:
Done